app/controllers/review_bids_controller.rb

```diff
@@ -40,36 +32,51 @@ def index
     render 'sign_up_sheet/review_bids_others_work'
   end
 
-  # provides variables for review bidding page
+  # computes important topic sets for displaying bids.
```

Comment:
These comments are repeated twice.

Reply:
Also, the variables are explained in random order.  I suggest alphabetical ordering.